bin/dotd

```diff
@@ -337,7 +306,7 @@ def main
     end
   end
 
-  do_dtt_subroutine
+  dtt_subroutine
```

Comment:
It was time a long time ago. :)

Though to be fair, the length of the script keeps decreasing, as more and more gets pulled out (e.g., `InfraTestTopic`, `GitHub`, etc.) and tested. That said, I'll follow this PR with some unit tests.